pkgs/games/openspades/default.nix

```diff
@@ -19,6 +20,15 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     freetype SDL2 SDL2_image libGL zlib curl glew opusfile openal libogg
+  ] ++ stdenv.lib.optionals stdenv.hostPlatform.isDarwin [
+    Cocoa
+  ];
+
+  patches = [
+    (fetchpatch {
```

Comment:
Could you add a comment here? As a general policy I think every patch should have at least one line explaining its purpose and (if applicable) its source.

Reply:
Good idea. Done.